sdk/monitor/Azure.Monitor.OpenTelemetry/README.md

```diff
@@ -1,19 +1,53 @@
 # Azure Monitor Distro client library for .NET
 
+The Azure Monitor Distro is a client library that sends telemetry data to Azure Monitor following the OpenTelemetry Specification. This library can be used to instrument your ASP.NET Core applications to collect and send telemetry data to Azure Monitor for analysis and monitoring, powering experiences in Application Insights.
+
 ## Getting started
 
 ### Prerequisites
 
+- **Azure Subscription:**  To use Azure services, including Azure Monitor Distro, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://azure.microsoft.com/account).
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net#find-your-connection-string).
```

Comment:
consider adding a prerequisite:
an asp.net core app. You can bring you own, or link to hello world asp.net core tutorial from asp.net docs.

Reply:
this is important to make it clear this is specific for asp.net core. (and not console or ASP.NET)